nodejs.org/locale/uk/foundation/board.md

```diff
@@ -1,23 +1,20 @@
 ---
-title: Node.js Foundation Board of Directors
+title: Рада директорів Node.js Foundation
 layout: foundation.hbs
 ---
 
-The Board of Directors is composed of representatives from corporate members, a representative of the
-Technical Steering Committee, and representatives elected by the individual membership class.
+Рада директорів складається з представників від корпоративних учасників, представників Технічного виконавчого комітету (Technical Steering Committee (TSC)), та представників, що обираються окремими учасниками.
 
-* Chairperson: **Danese Cooper**, distinguished member of technical staff - open source at PayPal.
-* Vice-Chairperson: **Scott Hammond**, chief executive officer at Joyent.
-* Secretary: **Gianugo Rabellino**, senior director of open source programs at Microsoft.
-* TSC Director: **Rod Vagg**, chief node officer at NodeSource.
-* Gold-level Director: **Charlie Robbins**, director of engineering at GoDaddy.com.
-* Silver-level Director: **Brian McCallister**, chief technology officer of platforms at Groupon.
-* Director: **Todd Moore**, vice president of open technology at IBM.
-* Director: **Rich Sharples**, senior director of product management at Red Hat.
-* Director: **Imad Sousou**, vice president and general manager at Intel.
-* Director: **Ashley Williams**, individual membership director.
-* Director: **Feross Aboukhadijeh**, individual membership director.
+* Голова: **Danese Cooper**, шановний член технічного персоналу - open source в PayPal.
+* Заступник голови: **Scott Hammond**, головний виконавчий директор (СTO) в Joyent.
+* Секретар: **Gianugo Rabellino**, старший директор open source програми в Microsoft.
```

Comment:
Out of date, see https://github.com/nodejs/nodejs.org/commit/16ef3de6109f01201e81c2e59566b99a588c8343

No Secretary right now, we will have board elections soon again for that.